js/radio-checkbox.js

```diff
@@ -20,9 +20,11 @@ CRM.$(function() {
         $this.attr('id', new Date().valueOf());
       }
 
-      if (!label.length) {
+      if (!label.length && !CRM.$('.page-civicrm-event').length) {
```

Comment:
Why we need to change this file?
Can't our functionality be achieved without this?

Reply:
Some of checkboxes were wrapped separately - without label, so they don't have styles, that's why I created custom labels.